packages/v4/src/content/design-guidelines/components/code-block/code-block.md

```diff
@@ -0,0 +1,34 @@
+---
+id: Code block
+section: components
+---
+
+The **code block** is a component that contains two or more lines of ready-only code. The code in a code block can be copied to the clipboard.
+
+## Elements
+Code block has four main components.
+
+<img src="./img/code-block-structure.png" alt="Elements of the code block component" width="441"/>
+
+1. **Code snippet:**  the content that will be copied to the user’s clipboard.
+2. **Clipboard button:** completes the copy action when clicked. The copy button is always present but additional actions can be added on an as needed basis.
```

Comment:
Wondering now if you should rename this to "Action buttons" or rename the heading underneath from "multiple actions" to "clipboards buttons" so that they match!

Could also hyperlink the 2nd and 4th bullet to the respective sections underneath, so that a user can jump to them directly. You would just go` [clipboard button](#clipboard-button)`
The string that comes after the # has to match the heading exactly (sentence cased), and the dashes equate to each space in the heading!

Reply:
I like the idea of calling them 'Action buttons' to keep it more general. 

Yup I can add the links to those sections. I'll reorder the paragraphs too so it follow the flow of the elements list.